### ISSUE TITLE
Isses with New "Labor"-Firmware

### ISSUE THREAD

**Issue by jk-andersen** [afuerhoff/ioBroker.fb-checkpresence#85]
**Describe the bug**  
Adapter ist not working after Upgrading to Labor / Beta
Log results:

<html>
<body>

fb-checkpresence.0 | 2022-02-23 21:35:11.982 | warn | {}
-- | -- | -- | --
fb-checkpresence.0 | 2022-02-23 21:35:11.981 | warn | {}
fb-checkpresence.0 | 2022-02-23 21:35:11.981 | warn | {}
fb-checkpresence.0 | 2022-02-23 21:35:11.979 | warn | {}
fb-checkpresence.0 | 2022-02-23 21:05:57.961 | warn | {}
fb-checkpresence.0 | 2022-02-23 21:05:57.959 | warn | {}
fb-checkpresence.0 | 2022-02-23 20:55:54.010 | warn | {}
fb-checkpresence.0 | 2022-02-23 20:55:54.009 | warn | {}
fb-checkpresence.0 | 2022-02-23 20:55:54.009 | warn | {}
fb-checkpresence.0 | 2022-02-23 20:55:54.007 | warn | {}
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..disabled" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..speed" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..active" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..macaddress" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..guest" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..interfacetype" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..active" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..speed" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..disabled" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..ipaddress" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..interfacetype" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.867 | warn | State "fb-checkpresence.0.fb-devices..guest" has no existing object, this might lead to an error in future versions
fb-checkpresence.0 | 2022-02-23 20:51:54.866 | warn | State "fb-checkpresence.0.fb-devices..ipaddress" has no existing object, this might lead to an error in future versions

</body>
</html>

**Fritzbox**  
6591 - Labor Version: 07.39-94697

**Versions:**  
 - Adapter version: v1.1.13
 - JS-Controller version: 4.0.15
 - Node version: v14.19.0
 - Operating system: Ubuntu LTS 20.04

**Comment by gitknd** [comment]
same here

**Comment by afuerhoff** [comment]
Hello, there is a device in the log with no name. That is not allowed. Please check in the fritzbox.
Please disable the adapter and reboot the fritzbox. Switch to loglevel debug, start the adapter and post the log again if there are any problems.

**Comment by jk-andersen** [comment]
Hi,

I digged a little deeper as it still does not work. Here is the debug:

<html>
<body>


fb-checkpresence.0 | 2022-03-10 12:25:25.323 | debug | getDeviceInfo activeCnt: 15
-- | -- | -- | --
fb-checkpresence.0 | 2022-03-10 12:25:25.045 | debug | loop main ends after 0,160089558 s
fb-checkpresence.0 | 2022-03-10 12:25:22.883 | debug | loop family ends after 0,383477988 s
fb-checkpresence.0 | 2022-03-10 12:25:22.882 | debug | history Jan cntHistory: 0
fb-checkpresence.0 | 2022-03-10 12:25:22.859 | debug | history cntActualDay: 9
fb-checkpresence.0 | 2022-03-10 12:25:22.821 | debug | history Rike cntHistory: 0
fb-checkpresence.0 | 2022-03-10 12:25:22.797 | debug | history cntActualDay: 7
fb-checkpresence.0 | 2022-03-10 12:24:24.670 | debug | getDeviceInfo activeCnt: 15
fb-checkpresence.0 | 2022-03-10 12:24:24.465 | debug | loop main ends after 0,159344641 s
fb-checkpresence.0 | 2022-03-10 12:24:22.305 | debug | loop family ends after 0,392081034 s
fb-checkpresence.0 | 2022-03-10 12:24:22.304 | debug | history Jan cntHistory: 0
fb-checkpresence.0 | 2022-03-10 12:24:22.279 | debug | history cntActualDay: 9
fb-checkpresence.0 | 2022-03-10 12:24:22.234 | debug | history Rike cntHistory: 0
fb-checkpresence.0 | 2022-03-10 12:24:22.202 | debug | history cntActualDay: 7
fb-checkpresence.0 | 2022-03-10 12:24:20.911 | info | loop successfully started
fb-checkpresence.0 | 2022-03-10 12:24:20.910 | info | states successfully subscribed
fb-checkpresence.0 | 2022-03-10 12:24:20.910 | info | createFbDeviceObjects finished successfully
fb-checkpresence.0 | 2022-03-10 12:24:19.562 | info | createMemberObjects finished successfully
fb-checkpresence.0 | 2022-03-10 12:24:19.403 | info | createGlobalObjects finished successfully
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | Layer3Forwarding1-GetDefaultConnectionService is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | LANConfigSecurity1-X_AVM-DE_GetCurrentUser is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | WANPPPConnection1-ForceTermination is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | DeviceConfig1-Reboot is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | X_AVM-DE_HostFilter-GetWANAccessByIP is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | X_AVM-DE_HostFilter-DisallowWANAccessByIP is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.326 | info | DeviceInfo1-GetInfo is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WLANConfiguration3-GetSecurityKeys is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WLANConfiguration3-GetInfo is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WLANConfiguration3-SetEnable is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WANIPConnection1-GetInfo is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WANPPPConnection1-GetInfo is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | WANCommonInterfaceConfig1-GetCommonLinkProperties is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | DeviceInfo1-GetSecurityPort is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | Hosts1-X_AVM-DE_GetSpecificHostEntryByIP is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | Hosts1-GetSpecificHostEntry is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | Hosts1-X_AVM-DE_GetMeshListPath is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.325 | info | Hosts1-X_AVM-DE_GetHostListPath is supported
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | info | configuration default connection: 1.WANIPConnection.1
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration filter delay: 15
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration guest info: true
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration qr code: false
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration ssl: true
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration compatibility: false
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration whitelist: false
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration mesh info: true
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration fb-devices true
fb-checkpresence.0 | 2022-03-10 12:24:19.324 | debug | configuration familymembers: [{"enabled":true,"group":"","familymember":"Rike","devicename":"FriederesiPhone","macaddress":"B6:EB:F7:XXXXX","ipaddress":"192.168.XXXXXX","usage":"MAC","usefilter":false,"comment":""},{"enabled":true,"group":"","familymember":"Jan","devicename":"Jan-ist-Pr0","macaddress":"A0:FB:C5:XXXXXXX","ipaddress":"192.168.XXXXXXX","usage":"MAC","usefilter":false,"comment":""}]
fb-checkpresence.0 | 2022-03-10 12:24:19.323 | debug | configuration familymembers count: 2
fb-checkpresence.0 | 2022-03-10 12:24:19.323 | debug | configuration dateformat: <yyyy.mm.dd HH:MM:ss>
fb-checkpresence.0 | 2022-03-10 12:24:19.323 | debug | configuration history: <influxdb.0>
fb-checkpresence.0 | 2022-03-10 12:24:19.323 | debug | configuration user: <checkpresence>
fb-checkpresence.0 | 2022-03-10 12:24:19.323 | info | start fb-checkpresence.0: FRITZ!Box 6591 Cable version: 161.07.39-94697 ip-address: "192.168.178.1" - interval devices: 1 min. - interval members: 60 s
fb-checkpresence.0 | 2022-03-10 12:24:18.995 | info | starting. Version 1.1.13 in /opt/iobroker/node_modules/iobroker.fb-checkpresence, node: v14.19.0, js-controller: 4.0.19


</body>
</html>


The adapter does not throw errors - It just does not recognize the devices active. (Replaced MAC-Addresses and IPs with XXXX)

In my Fritzbox - only some (15) devices are shown as "active" - The majority of devices (30+ including the checked phones here) are shown inactive in the "network-tab"- The reason is unknown for me.
I submitted a bug-report to AVM, did not get any reply, yet.

**Comment by afuerhoff** [comment]
@jk-andersen 
Is it possible that you have configured a private mac address on your cellphone? If this is the case use the hostname setting for the family member. Just to rule that out.

**Comment by jk-andersen** [comment]
Thanks @afuerhoff, I ran into that with iOS14 first release a year ago - It is not the issue now.

I got feedback from AVM - Currently the "Labor / beta" hat issues with the recognition of "active devices" with (some) switches in the Network. In my case there are several Zyxel Managed switches with Multicast enabled. They are adressing this in the next Labor-Version - For me this seems likle the root-cause.
I will update you then :)
